data/backup.yml

```diff
@@ -66040,7 +66040,7 @@ attributes:
       \r\n\
       Or watch a longer excerpt of Leslie's talk on ABC News 24. Stay tuned for specific dates and times when the program will go to air. "
     link: ""
-    publish_on: April 01, 2011
```

Comment:
Altering the input file to match your input requirements should be avoided. It's better to either process the data on the fly into the format you want or write out a new file for input into the next stage. While the file is in version control so you can always go back and see what you have done it is better to keep the supplied source around so you can always easily see what the original content looked like.